core/src/main/java/com/linecorp/armeria/common/AttributesGetters.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common;
+
+import java.util.Iterator;
+import java.util.Map.Entry;
+
+import com.linecorp.armeria.common.annotation.Nullable;
+import com.linecorp.armeria.common.annotation.UnstableApi;
+
+import io.netty.util.AttributeKey;
+
+/**
+ * Provides the getter methods to {@link Attributes} and {@link AttributesBuilder}.
```

Comment:
```suggestion
 * Provides the getter methods to {@link Attributes} and {@link ConcurrentAttributes}.
```

Reply:
It is cruft. I should've cleaned up correctly.